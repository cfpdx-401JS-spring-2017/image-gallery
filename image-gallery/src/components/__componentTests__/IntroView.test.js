import React from 'react';
import ReactDOM from 'react-dom';
import Intro from '../IntroView';
import renderer from 'react-test-renderer';




it('renders List view without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Intro/>, div);
});

it('renders a snapshot', () => {
  const tree = renderer.create(<Intro/>).toJSON();
  expect(tree).toMatchSnapshot();
});