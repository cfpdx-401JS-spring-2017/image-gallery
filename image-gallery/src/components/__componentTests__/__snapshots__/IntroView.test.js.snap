// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="intro-image brighten"
>
  <a
    className="video"
    href="https://www.youtube.com/watch?v=cKC-XBwQ_pA"
  >
    <img
      alt="standing on boards"
      src="introImage.jpg"
    />
  </a>
  <span
    className="play-video"
  >
    <h1>
      PLAY VIDEO
    </h1>
  </span>
</div>
`;
